lib/prepare.js

```diff
@@ -35,6 +36,7 @@ const FileUpdater = require('cordova-common').FileUpdater;
 const projectFile = require('./projectFile');
 const Podfile = require('./Podfile').Podfile;
 const check_reqs = require('./check_reqs');
+const ConfigParseriOS = require('./ConfigParseriOS');
```

Comment:
If class name changes, update here as well.

Reply:
Yes I updated it.